markhor_description/urdf/markhor_w_attachments.xacro

```diff
@@ -2,33 +2,15 @@
 
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="markhor">
   <!-- Include markhor robot -->
-  <xacro:include filename="$(find markhor_description)/urdf/markhor.xacro"/>
-
-  <link name="markhor_root"/>
-  <xacro:property name="robot_root" value="markhor_root" />
-  <xacro:markhor base_parent="${robot_root}"/>
```

Comment:
Those 3 lines are no longer needed since we changed the structure of markhor.xacro